deployment-docs/README.md

```diff
@@ -0,0 +1,63 @@
+# Deployment Documentation
+
+*NOTE*: This is a work in progress, as we develop processes for flexibly and reliably deploying Taskcluster.
+
+## Overview
+
+A Taskcluster deployment is identified by a rootUrl, which defines the domain name on which the deployment can be accessed.
+
+Taskcluster is deployed as a collection of interacting microservices within a Kubernetes cluster.
+The cluster has an [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) that routes various paths under the rootUrl to the appropriate services.
+The `/` path is routed to the taskcluster-ui service, which serves a browser-based user interface.
+
+The services all run from a single Docker image, referred to as the "monoimage", that is built by running `yarn build` in this repository.
+
+## Terraform
+
+Taskcluster is deployed using Terraform.
+The Terraform configuration is in `infrastructure/terraform` and consists of a Terraform module that should be included into your own Terraform configuration, providing the necessary configuration.
+See the `variables.tf` file for a list of required values.
+
+The Terraform module contains a [reference](../infrastructure/terraform/taskcluster.tf.json) to the Docker image that will be deployed.
+This is updated periodically in the repository; a more formal approach is in the design stages.
+
+## Clouds
+
+Taskcluster is a multi-cloud system, and as such expects credentials for several cloud providers.
```

Comment:
This is a very positive way to spin this. :100:

Reply:
"Taskcluster makes great lemonade, so it ships with a large quantity of lemons"